test/vcr_python_wrapper.py

```diff
@@ -81,10 +81,13 @@ def host_body_matcher(r1, r2):
     # Call the original python script with vcr-cassette in place
     fam_vcr = vcr.VCR()
 
-    query_matcher = 'query'
     if test_params['test_name'] == 'domain':
         fam_vcr.register_matcher('domain_query', domain_query_matcher)
         query_matcher = 'domain_query'
+    else:
+        query_matcher = 'query'
+
+    fam_vcr.register_matcher('body_json_l2', body_json_l2_matcher)
```

Comment:
We do not need to change that file here at all.

Reply:
I have removed the changes in vcr_python_wrapper file.